### ISSUE TITLE
Please add LICENSE

### ISSUE THREAD

**Issue by twardoch** [metafloor/freetype-js#1]
@metafloor : Could you add a copy of https://github.com/metafloor/bwip-js/blob/master/LICENSE.FREETYPE to this repo, to ensure that the custom files are licensed under the same terms as FreeType itself? It would help people who’d like to use it :)

**Comment by metafloor** [comment]
Not sure what you are asking...  The license file for freetype is already part of the bwip-js dist.

**Comment by twardoch** [comment]
Yes, I'm asking to clarify the licensing of this repo (freetype-js), i.e. the build scripts for freetype.js itself. :)

**Comment by twardoch** [comment]
BTW, the bwip-js demo looks great and I'm trying to convince some people to make a general-purpose font viewer for FreeType that'd run in the browser — largely because FreeType supports some font formats that aren't supported by the browsers yet. 

Also, it could be combined with https://github.com/prezi/harfbuzz-js to get functionality comparable with `hb-view` (a font viewer that works with Harfbuzz) combined with `ft-view` and `ft-multi` (parts of the ft2demos package). It’d be super-useful for font developers. 

Effectively, it could be something like http://www.impallari.com/testing/ but using Harfbuzz.js and FreeType.js to be fully platform independent.

**Comment by metafloor** [comment]
It's a monday and I'm only halfway through my first cup of coffee.  Wasn't seeing that the issue you raised was on the freetype repo.  The license is there now.

Also, thanks for the feedback.  I am using freetype-js on another project where trying to use custom fonts with standard canvas has too many hoops you have to jump through to get them to render cross browser.   With freetype-js, I just xhr down the font and start using it.
